### ISSUE TITLE
[BUG] After successfully running an action, the action modal is not closed and neither the success nor failure message appears

### ISSUE THREAD

**Issue by drewprice** [avo-hq/avo#506]
### Describe the bug
When an action is run, although the request is handled (i.e. does not 500), the action modal stays in place, the action  responses (from `succeed` or `fail`) do not appear (though something does flicker behind the modal), and the resource is not reloaded.

Manually refreshing the page reveals that the action _did_ run (as the application logs also confirm).

### Steps to Reproduce
1. Go to a resource with an action
2. Select and run the action
3. The modal stays in place, no response message appears, the resource is not reloaded

### Expected behavior
The modal should clear, the response message should appear, and the resource should be reloaded.

### Actual behavior
The modal stays fixed, no response message appears, the resource is not reloaded (after manually closing the modal)

### System configuration
**Avo version**: 1.13.1

**Rails version**: 6.1.3

**Ruby version**: 2.6.6

### Screenshots

https://user-images.githubusercontent.com/7977907/132390608-bc040762-6fe1-4b79-bfee-7c0f7dd476b2.mov

### Additional context

The action object used in the above screen recording:

```ruby
# frozen_string_literal: true

class CanvasDeactivateUser < Avo::BaseAction
  self.name = 'Deactivate User'
  self.message = <<~MSG
    Are you sure you want to deactivate this user?
    This will action will deactivate each of their Canvas enrollments, but the user will not be removed from Canvas.
  MSG

  self.confirm_button_label = 'Deactivate'

  def handle(models:, **_rest)
    models.each(&:deactivate_externally!)

    succeed 'Done!'
  rescue => e
    self.fail "An error occurred; #{e}"
  end
end
```

In addition to the action in the recording above, I've created simpler actions and attached them to other resources with the same results. e.g.

```ruby
def handle(models:, fields:)
  succeed 'Hello, world!'
end
```

**Comment by adrianthedev** [comment]
Hi 👋

Hmmm... Could you please try updating to the latest version?
Also, what response are you getting from the action submission? You should receive a response with a few turbolinks instructions (screenshot below).

![image](https://user-images.githubusercontent.com/1334409/132493697-b3e1e13e-db9d-4ab7-a2e1-3129978402c6.png)

You can see the way it works [here](https://avodemo.herokuapp.com/avo/resources/users/39).

https://github.com/avo-hq/avo/issues/505 might have the same issue. I'd like to fix this one and see if this fixes the other one too.

**Comment by drewprice** [comment]
Hey @adrianthedev! Thanks for the quick response.

I've double checked that I'm on the latest Avo (1.13.1), and I've also bumped my Rails version to the latest (so I'm now on 6.1.4).

The response definitely seems strange, especially after playing around with the demo (thanks for that link 🙏🏻). Rather than the 200 response with turbolinks that I see in the demo, I'm getting a 302:

https://user-images.githubusercontent.com/7977907/132558376-09b1fdc5-2fda-4c4c-9187-220c5e51f258.mov

**Comment by adrianthedev** [comment]
Hmm... I remember somebody else had a similar issue. The culprit for him was that he updated rails form 6.0 to 6.1 and forgot to update the `config.load_defaults 6.1` in `application.rb`.
Could you confirm first if this is the case here?

Also... one more important detail is: Did the actions work for this app ever? So at one point, they stopped working?

**Comment by drewprice** [comment]
![tenor-254060629](https://user-images.githubusercontent.com/7977907/132559806-8973c071-30c2-49fe-af6e-ef01aa4c6ee4.gif)

That was exactly it! 🤦🏻 Thank you, @adrianthedev!

This was the first action we've added, so no prior history to base this on. But yeah, we needed to bump `config.load_defaults 6.1`, and now that we've done that, we're no longer seeing the issue (we are still seeing #505 though).
